gatorgrade/output/tests/test_function.py

```diff
@@ -0,0 +1,60 @@
+""" Tests to ensure the output_functions.py functions work properly. """
+
+import pytest
+from colorama import init, Fore
+from output import output_functions
+
+init()
+
+
+@pytest.fixture()
+def test_output_shows_green(capsys):
+    """Test for ouput of a passed check will show green check."""
+    output_functions.output_passed_checks()
+    out, err = capsys.readouterr()
+
+    assert f"{Fore.GREEN}\u2714" in out
+    assert err == ""
+
+
+def test_output_shows_red(capsys):
+    """Test for output of failed check to show red color using Colorama."""
+    output_functions.output_failed_checks()
+    out, err = capsys.readouterr()
+
+    assert f"{Fore.RED}" in out
+    assert err == ""
+
+
+def test_output_shows_yellow(capsys):
+    """Testing the failed check will show a yellow message in output."""
+    output_functions.output_failed_checks(
+        failed_checks=("Implement this with an if.", False, "No if statements found")
+    )
+    out, err = capsys.readouterr()
+
+    assert f"{Fore.YELLOW}\u2192" in out
+    assert err == ""
+
+
+def test_descrition_in_fail_message(capsys):
+    """Testing the failed check will show diagnostic yellow message in output."""
+    output_functions.output_failed_checks(
+        failed_checks=("Implement this with an if.", False, "No if statements found")
+    )
+    out, err = capsys.readouterr()
+    expected_output = "No if"
+    actual_output = out
+    assert expected_output in actual_output
+    assert "" in err
+
+
+def test_false_result_returns_x(capsys):
+    """Test for the X will appear in the output for the failed check."""
+    output_functions.output_failed_checks(
+        failed_checks=("Implement this with an if.", False, "No if statements found")
+    )
+    out, err = capsys.readouterr()
+
+    assert "\u2718" in out
```

Comment:
In this test you asserted against out, while in the previous one you reassigned out to 'actual_output'. Would you mind making this a uniform choice?

Reply:
I made the change to assert actual output for all, thank you.